### PR TITLE
[plugin-mobile-app] Add step to upload file to a device

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 1]
```diff
@@ -102,6 +102,26 @@ a|Defines whether the tests run on real device
 
 !===
 
+|`mobile-application.device-folder`
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:105]: `mobile-environment.device.folder-for-file-uploading`

File: vividus-plugin-mobile-app/build.gradle [hunk 2]
```diff
@@ -13,4 +13,6 @@ project.description = 'Vividus plugin for testing mobile applications'
      testImplementation(group: 'org.mockito', name: 'mockito-core', version: versions.mockito)
      testImplementation(group: 'org.mockito', name: 'mockito-junit-jupiter', version: versions.mockito)
      testImplementation(group: 'com.github.valfirst', name: 'slf4j-test', version: versions.slf4jTest)
+
+     testCompile(group: 'com.github.spotbugs', name: 'spotbugs-annotations', version: spotbugsVersion)
```
Comment (valfirst) [inline vividus-plugin-mobile-app/build.gradle:17]: `testCompileOnly`

**Review by ikalinin1 — APPROVED** [review]

File: vividus-plugin-mobile-app/src/main/resources/org/vividus/mobileapp/action/data.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+:)
```
Comment (valfirst) [inline vividus-plugin-mobile-app/src/main/resources/org/vividus/mobileapp/action/data.txt:1]: what for?

File: vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story [hunk 4]
```diff
@@ -70,3 +70,28 @@ Then number of elements found by `accessibilityId(<endElementAccessibilityId>)`
 When I swipe DOWN to element located `accessibilityId(<startElementAccessibilityId>)` with duration PT1S
 Then number of elements found by `accessibilityId(<startElementAccessibilityId>)` is equal to `1`
 Then number of elements found by `accessibilityId(<endElementAccessibilityId>)` is equal to `0`
+
+
+Scenario: [Android] Verify step: 'When I upload file `$filePath` to device'
+Meta: @targetPlatform android
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story:76]:
```
Meta:
    @targetPlatform android
```

File: vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story [hunk 5]
```diff
@@ -70,3 +70,28 @@ Then number of elements found by `accessibilityId(<endElementAccessibilityId>)`
...
+When I tap on element located `accessibilityId(select-image-accessibilityLabel)`
+When I tap on element located `xpath(//android.widget.TextView[@text='Choose from Library…'])`
+When I tap on element located `xpath(//android.widget.Button[@text='Allow'])`
+When I tap on element located `xpath(//android.widget.TextView[@text='Pictures'])`
+When I tap on element located `xpath((//android.view.ViewGroup[contains(@content-desc, "Photo taken")])[1])`
+Then number of elements found by `xpath(//android.widget.TextView[@text='228x228'])` is equal to `1`
+
+
+Scenario: [iOS] Verify step: 'When I upload file `$filePath` to device'
+Meta: @targetPlatform ios
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story:89]:
```
Meta:
    @targetPlatform ios
```

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-plugin-mobile-app/src/main/resources/org/vividus/mobileapp/action/data.txt (hunk 3 above)
Comment (uarlouski) [reply vividus-plugin-mobile-app/src/main/resources/org/vividus/mobileapp/action/data.txt]: moved to test package

**Review by valfirst — APPROVED** [review]

**Pull request merged**